README.md

```diff
@@ -58,10 +58,15 @@ recipient = client.recipient.find('R-1234567abcdefg')
 print recipient.id
 ```
 
-#### Need a proxy?
+#### Configuring a proxy
 
 ```Ruby
-client = Trolley.client('YOUR-API-KEY', 'YOUR-SECRET-KEY', 'production', proxy_uri: 'peter_the_proxy.com')
+client = Trolley.client('YOUR-API-KEY', 'YOUR-SECRET-KEY', proxy_uri: 'peter_the_proxy.com')
```

Comment:
Lets change:
'YOUR-API-KEY' -> 'ACCESS-KEY'
'YOUR-SECRET-KEY' -> 'SECRET-KEY'

Some customers would keep searching for `API-KEY` or `SECRET-KEY` in dashboard and never find it, so this will keep it consistent with documentation everywhere.
(Drawing from our experience with webhooks support queries)